bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -90,6 +141,11 @@ export async function getDependentStoryFiles(
   ].map(normalize);
 
   const modulesByName: Record<string, Module> = {};
+  // stores the normalized paths (like modulesByName does), but groups all package files under the same package name.
+  // Why? In monorepo, may have multiple node_modules directories, but we want to update any components using an updated dependency
+  // (regardless of where it's used). So we use the package name as the key, and the value is all files of that package name,
+  // regardless of where they are.
+  const thirdPartyModules: Record<string, string[]> = {};
```

Comment:
I find this comment a bit confusing, but I think I understand the point based on how it's used below. This is mapping a package name (e.g. `@storybook/addon-actions`) to all the module names we find that we think are part of that package.

I wonder if maybe using some type aliases (so instead of `Record<string, string[]>` it's `Record<PackageName, NormalizedModulePath[]>` or something), might make this easier to understand at a glance.